internal/experiment/dnscheck/dnscheck.go

```diff
@@ -317,16 +317,3 @@ func NewExperimentMeasurer(config Config) model.ExperimentMeasurer {
 		Endpoints: nil, // disabled by default
 	}
 }
-
-// SummaryKeys contains summary keys for this experiment.
-//
-// Note that this structure is part of the ABI contract with ooniprobe
-// therefore we should be careful when changing it.
-type SummaryKeys struct {
-	IsAnomaly bool `json:"-"`
-}
-
-// GetSummaryKeys implements model.ExperimentMeasurer.GetSummaryKeys.
-func (m *Measurer) GetSummaryKeys(measurement *model.Measurement) (interface{}, error) {
-	return SummaryKeys{IsAnomaly: false}, nil
-}
```

Comment:
Now we implicitly provide the correct always-returning-false implementation when an experiment does not implement the `MeasurementSummaryKeysProvider` interface. So, we can safely zap the boilerplate code concerning the summary keys for each experiment that always returned false.